src/diagram/models/mathjs-utils.test.ts

```diff
@@ -0,0 +1,38 @@
+import { replaceInputNames } from "./mathjs-utils";
+
+describe("mathjs-utils", () => {
+  describe("replaceInputNames", () => {
+    it("replaces backtick variables", () => {
+      const result = replaceInputNames("`var`", ["var"]);
+      expect(result).toBe("input_0");
+    });
+
+    it("replaces backtick variables with multiple inputs", () => {
```

Comment:
It shouldn't really matter, but since the primary use case for backtick variables is names with spaces there would be some illustrative value in demonstrating that in a test.